src/components/vehicleReference/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  AspectRatio,
+  Box,
+  chakra,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  renderImage: ReactNode;
+  carTitle: string;
+  price: string;
+  dealerName: string;
```

Comment:
Shouldn't this be:
```suggestion
  sellerName: string;
```
?

Reply:
😅 yes!